ExampleApp/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "react": "16.11.0",
     "react-native": "0.62.2",
-    "react-native-jitsi-meet": "^2.1.1"
+    "react-native-jitsi-meet": "RocketChat/react-native-jitsi-meet#update-lib"
```

Comment:
This is weird.
This repo doesn't use [bob](https://github.com/callstack/react-native-builder-bob), so example app doesn't work based on local files and need changes to be pushed to be tested.
We might want to create our own version of it.